### PR TITLE
done

### PR DESCRIPTION
**Pull request by Mohamed9820m**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -829,3 +829,194 @@ ul {
 }
 
 /* 3rd subpage */
+
```
Comment (basiacarvalho) [inline css/style.css:832]: There are two empty lines here. One is enough

File: css/style.css [hunk 2]
```diff
@@ -829,3 +829,194 @@ ul {
...
+
+body {
+  font-family: 'Roboto-medium', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
```
Comment (basiacarvalho) [inline css/style.css:839]: Many empty lines. Please have a look at the rest of the code, I won't write any more comments regarding these empty lines

File: pages/blog.htm [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+              <div class="blog-card">
+                <img src="../images/javascript.jpg" alt="Blog Image">
+                <h3>Blog Title </h3>
+                <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat</p>
+              </div>
+    
+    
+            </div>
+          </section>
+
```
Comment (basiacarvalho) [inline pages/blog.htm]: There are many empty lines in this file. Please remove them.

**Review by basiacarvalho — CHANGES_REQUESTED** [review]

**Comment by basiacarvalho** [comment]
Hi,

I've just finished checking Lezanders pull request and I had another look at yours. 
Unfortunately, there are some issues and conflicts to take care of that we don't have time for now.
Apart from that, I checked from your local branch that it changes the appearance of other pages.
I've just talked to Elisabeth as Lezanders is not available, and we decided to not merge your pull request now for the benefit of the whole group. During the presentation, I will let Branko know that your pull request is open.

Basia

**Comment by Mohamed9820m** [comment]
i fix all the issues that you send them what is missing !


Le sam. 3 févr. 2024 à 09:22, basiacarvalho ***@***.***> a
écrit :

**Pull request closed without merging**